doc/releases/pacific.rst

```diff
@@ -2,6 +2,240 @@
 Pacific
 =======
 
+v16.2.15 Pacific
+================
+
+This is the fifteenth, and expected to be last, backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* `ceph config dump --format <json|xml>` output will display the localized
+  option names instead of their normalized version. For example,
+  "mgr/prometheus/x/server_port" will be displayed instead of
+  "mgr/prometheus/server_port". This matches the output of the non pretty-print
+  formatted version of the command.
+
+* CephFS: MDS evicts clients who are not advancing their request tids, which causes
+  a large buildup of session metadata, resulting in the MDS going read-only due to
+  the RADOS operation exceeding the size threshold. The `mds_session_metadata_threshold`
+  config controls the maximum size that an (encoded) session metadata can grow.
+
+* RADOS: The `get_pool_is_selfmanaged_snaps_mode` C++ API has been deprecated
+  due to its susceptibility to false negative results.  Its safer replacement is
+  `pool_is_in_selfmanaged_snaps_mode`.
+
+* RBD: When diffing against the beginning of time (`fromsnapname == NULL`) in
+  fast-diff mode (`whole_object == true` with `fast-diff` image feature enabled
+  and valid), diff-iterate is now guaranteed to execute locally if exclusive
+  lock is available.  This brings a dramatic performance improvement for QEMU
+  live disk synchronization and backup use cases.
+
+Changelog
+---------------
```

Comment:
@yuriw I think the formatting matters here.
```suggestion
Changelog
---------
```

Reply:
done